huaweicloud/utils/utils.go

```diff
@@ -386,3 +387,31 @@ func RandomString(n int, allowedChars ...[]rune) (result string) {
 	result = string(b)
 	return
 }
+
+// OBSLogIsDebugOrHigher returns a bool type parameter, which specifies whether to print to OBS log
+func OBSLogIsDebugOrHigher() bool {
+	providerLogLevel := os.Getenv("TF_LOG_PROVIDER")
+	if providerLogLevel == "" {
+		return logging.IsDebugOrHigher()
+	}
+
+	logLevel := "TRACE"
```

Comment:
the init value should be "INFO"?

Reply:
if we make it "INFO", the behavior func is different from the original func

Reply:
rewrite this func, only print log when the log level id valid